src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/MSSQL/0023-View-StudentProgramDim-Create.sql

```diff
@@ -0,0 +1,34 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+
+IF EXISTS
+(
+    SELECT 1
+    FROM [INFORMATION_SCHEMA].[VIEWS]
+    WHERE [TABLE_SCHEMA] = 'analytics'
+          AND [TABLE_NAME] = 'StudentProgramDim'
+)
+    BEGIN
+        DROP VIEW [analytics].[StudentProgramDim];
+END;
+GO
+CREATE VIEW [analytics].[StudentProgramDim]
+AS
+    SELECT [BeginDate],program.[EducationOrganizationId],program.[ProgramName],student.[StudentUSI],
+    CONCAT (Student.StudentUniqueId,'-',StudentSchoolAssociation.SchoolId) AS StudentSchoolKey
+    FROM [edfi].[Program]
+    INNER JOIN
+        [edfi].[StudentProgramAssociation] StudentProgram
+        ON StudentProgram.ProgramName = Program.ProgramName
+        AND StudentProgram.ProgramTypeDescriptorId = Program.ProgramTypeDescriptorId
```

Comment:
If I comment out this line, test are not failing. More rows are needed to validate this JOIN (e.g. Add a new program with different ProgramTypeDescriptorId, but with same ProgramName and EducationOrganizationId of other row.